webauthn/helpers/structs.py

```diff
@@ -19,25 +18,27 @@ class WebAuthnBaseModel(BaseModel):
     - Encodes bytes to Base64URL
     - Converts snake_case properties to camelCase
 
-    `Model.parse_raw()` (from JSON):
+    `Model.model_validate_json()` (from JSON):
```

Comment:
Does the reference to `modelInstance.json()` above need to get updated too to mention `model_dump_json()` instead?